src/basic_bot/debug/test_ai-edge-litert.py

```diff
@@ -58,7 +58,7 @@
 
 print("reading test image")
 img = cv2.imread(test_image)
-initial_h, initial_w, channels = img.shape
+initial_h, initial_w, channels = img.shape  # type: ignore
```

Comment:
I still haven't figured out why same version of code, same version of mypy and same mypy.ini, but different errors on other platforms.  This line and one changed below where complaining about numpy array doesn't have a `.shape`.   I think maybe older type defs for opencv??.